main.go

```diff
@@ -257,10 +323,35 @@ func connectAction(ctx *cli.Context) error {
 		durations[BrandName] = time.Since(start)
 	}
 
-	/* 4. Show footer message */
+	/* 4. Show Configuration Profile */
+	if len(errorMessages) == 0 {
+		// Update the configuration file
+		config.CADir, _ = getCADir()
+		// Generate a new http client configured with mutual TLS certificate
+		tlsConfig, err := config.CreateTLSClientConfig()
+		if err != nil {
+			errorMessages[BrandName] = fmt.Errorf("cannot configure TLS: %w", err)
+		}
+		client, err := http.NewHTTPClient(tlsConfig)
+		if client != nil && err == nil {
```

Comment:
This is an unusual pattern. This could check `if err != nil` like most function calls. Then proceed to use the `client` without checking whether it's nil or not. It's a programmer error (and should cause a crash) if the function NewHTTPClient returns a nil client and a nil error, so it's OK to proceed assuming client is not nil after checking the value of err. That's a long-winded way of saying:

```suggestion
		if err != nil {
		    ...
		}
		profile, err := getConfProfile(client)
```

Reply:
You are totally right!